djangocms_rest_api/serializers/mapping.py

```diff
@@ -1,10 +1,14 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import, print_function, unicode_literals
 
-from cmsplugin_filer_image.cms_plugins import FilerImagePlugin
+from django.conf import settings
 
-from djangocms_rest_api.serializers.filerimageplugin_serializer import FilerImagePluginSerializer
 
-serializer_class_mapping = {
```

Comment:
I did that so we can use serializer from parent if not defined and don't fall back to auto generated, if you think we don't need that I'll remove that.